src/components/cc-header-app/cc-header-app.js

```diff
@@ -208,58 +199,57 @@ export class CcHeaderApp extends LitElement {
       this._lastUserAction = null;
     }
   }
 
   render () {
 
     // Quick short circuit for errors
-    if (this.appInfoState.type === 'error') {
+    if (this.state.type === 'error') {
       return html`<cc-notice intent="warning" message="${i18n('cc-header-app.error')}"></cc-notice>`;
     }
 
-    const isAppInfoLoading = this.appInfoState.type === 'loading';
-    const appInfo = this.appInfoState.type === 'loaded' ? this.appInfoState : SKELETON_APP;
-    const status = this.statusAndCommitState.type === 'loaded' ? this.statusAndCommitState.status : SKELETON_STATUS;
+    const skeleton = this.state.type === 'loading';
+    const zoneState = this.state.type === 'loaded' ? { type: 'loaded', ...this.state.zone } : { type: 'loading' };
+    const appInfo = this.state.type === 'loaded' ? this.state : SKELETON_APP_INFO;
 
-    const isStatusAndCommitInfoLoading = this.statusAndCommitState.type === 'loading';
-    const isDeploying = ['restarting', 'restarting-with-downtime', 'starting'].includes(status);
-    const isRunning = ['restart-failed', 'restarting', 'running'].includes(status);
+    const isDeploying = ['restarting', 'restarting-with-downtime', 'starting'].includes(appInfo.status);
+    const isRunning = ['restart-failed', 'restarting', 'running'].includes(appInfo.status);
 
     return html`
      <div class="wrapper">
        <div class="main">
-         <div class="flavor-logo ${classMap({ skeleton: isAppInfoLoading })}" title=${ifDefined(appInfo.variantName)}>
+         <div class="flavor-logo ${classMap({ skeleton })}" title=${ifDefined(appInfo.variantName)}>
            <!-- image has a presentation role => alt="" -->
            <img class="flavor-logo_img" src=${ifDefined(appInfo.variantLogo)} alt="">
          </div>
 
          <div class="details">
-           <div class="name"><span class=${classMap({ skeleton: isAppInfoLoading })}>${appInfo.name}</span></div>
+           <div class="name"><span class=${classMap({ skeleton })}>${appInfo.name}</span></div>
            <div class="commits">
-             ${this._renderCommit(appInfo.commit, 'git', this.appInfoState.type === 'loading')}
+             ${this._renderCommit(appInfo.commit, 'git', this.state.type === 'loading')}
 
-             ${this.statusAndCommitState.type === 'loaded' && isDeploying
-             ? this._renderCommit(this.statusAndCommitState.startingCommit, 'starting', false)
-             : ''}
+             ${this.state.type === 'loaded' && isRunning
+                ? this._renderCommit(appInfo.runningCommit, 'running', false)
+                : ''}
 
-             ${this.statusAndCommitState.type === 'loaded' && isRunning
-             ? this._renderCommit(this.statusAndCommitState.runningCommit, 'running', false)
-             : ''}
+             ${this.state.type === 'loaded' && isDeploying
+                ? this._renderCommit(appInfo.startingCommit, 'starting', false)
+                : ''}
```

Comment:
The order change between `isRunning` & `isDeploying` is made on purpose:
When I checked the stories, I realized I was showing `starting` commits before `running` which was the opposite of what we had in prod. 
Now if you check the `restarting` story, it should be the same between preview & prod.